package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "@guardian/consent-management-platform",
-	"version": "6.0.0",
+	"version": "6.2.0-0",
```

i don't htink this needs to be in here, `np` should work it out?
```suggestion
	"version": "6.0.0",
```

---

will it include those changes into `6.0.0` once we merge them into main branch?

---

i was thinking to make another release for these with the latest changes, is it needed?